aperturedb/ParallelQuery.py

```diff
@@ -69,13 +69,30 @@ def execute_batch(q, blobs, db,
         # Transaction failed entirely.
         logger.error(f"Failed query = {q} with response = {r}")
         result = 1
-    if isinstance(r, dict) and db.last_response['status'] != 0:
-        logger.error(f"Failed query = {q} with response = {r}")
+
+    statuses = {}
+    if isinstance(r, dict):
+        statuses[r['status']] = [r]
+    elif isinstance(r, list):
+        # add each result to a list of the responses, keyed by the response
+        # code.
+        [statuses.setdefault(result[cmd]['status'], []).append(result)
```

Comment:
This is something that might increase in size for big ingestion runs or if people who run queries using this. To be kept an eye on.

Reply:
Oh you aggregate this from the caller. Should be OK.